ocs_ci/deployment/deployment.py

```diff
@@ -63,6 +67,12 @@ class Deployment(object):
     """
     Base for all deployment platforms
     """
+
+    # Default storage class for StorageCluster CRD,
+    # every platform specific class which extending this base class should
+    # define it
+    DEFAULT_STORAGECLASS = None
```

Comment:
Should this be an instance variable instead that we can set in the object `__init__`?

Reply:
This is done as a class attribute so that it will be defined even without instantiating a deployment class. This could be useful when some code want's to check a value of a default storage class on a particular platform we know how to deploy, without instantiating it's deployment class. We know this detail in advance, as part of the platform definition, after all.